code/game/objects/structures/xeno.dm

```diff
@@ -885,6 +888,8 @@ TUNNEL
 
 /obj/effect/alien/resin/acidwell/Crossed(atom/A)
 	. = ..()
+	if(CHECK_MULTIPLE_BITFIELDS(A.flags_pass, HOVERING))
+		return
```

Comment:
Why not small structs or such instead
this also goes for most of the other CHECK_MULTIPLE_BITFIELDS checks

Reply:
It's exactly the same performance wise, and it's more explicit. The flag PASSSLAMSTRUCT could be latter use by another item/mob, so why not making sure it's only jetpack that doesn't set trap?